apps/meteor/client/views/room/composer/messageBox/MessageBoxActionsToolbar/hooks/useFileUploadAction.ts

```diff
@@ -7,7 +7,7 @@ import { useChat } from '../../../../contexts/ChatContext';
 
 const fileInputProps = { type: 'file', multiple: true };
 
-export const useFileUploadAction = (disabled: boolean): GenericMenuItemProps => {
+export const useFileUploadAction = (disabled: boolean, handleFiles?: any): GenericMenuItemProps => {
```

Comment:
Please make the type correct, restricting to the right function signature

Reply:
handleFiles should not be optional since it's now required to make the upload work

Reply:
done sir and also made it compulsory term